runtime/protos/runtime.proto

```diff
@@ -256,34 +294,87 @@ message Profile {
   repeated string errors = 4;
 }
 
-// HealthStatus specifies the possible types of health status for a replica.
-enum HealthStatus {
-  UNKNOWN = 0;
-  HEALTHY = 1;
-  UNHEALTHY = 2;
-  TERMINATED = 3;
+// UpdateRoutingInfoRequest is a request from an envelope with the latest
```

Comment:
I would rename this comment to make it more "request"-y. This is one of the non-request requests from the envelope, so let's be more explicit:

```
// UpdateRoutingInfoRequest is a request from an envelope to the weavelet to update its routing information
// for a particular component.
```
```